configs/templates/_digitalocean.txt

```diff
@@ -32,6 +32,10 @@ CREDENTIALS = $DO_CREDENTIALS
 SSH_KEY_NAME = $DO_SSH_KEY_NAME
 KEY_NAME = $DO_KEY_NAME
 LOGIN = $DO_LOGIN
+NETNAME = private
+CAPTURE_SUPPORTED = $True
```

Comment:
so, for Digital Ocean, yes, I have implemented (and tested), the capture function.